commit 0ac62428bbacd352d5641b67826e360510a77541
Author: Bertram Felgenhauer <int-e@gmx.de>
Date:   Sat Mar 18 18:41:26 2017 +0100

    bump vector upper bound

diff --git a/safecopy.cabal b/safecopy.cabal
index f0ecf3c..45ebac3 100644
--- a/safecopy.cabal
+++ b/safecopy.cabal
@@ -39,7 +39,7 @@ Library
                        template-haskell < 2.12,
                        text < 1.3,
                        time < 1.7,
-                       vector >= 0.10 && < 0.12
+                       vector >= 0.10 && < 0.13
   if !impl(ghc > 8.0)
     Build-Depends:     semigroups >= 0.18 && < 0.19
 
